resources/sass/pages/_plp.scss

```diff
@@ -0,0 +1,214 @@
+        &:after {
+            content: "";
+            display: block;
+            height: 1px;
+            width: 100%;
+            margin: 10px 0px;
+            background-color: #eee;
+            text-decoration: none;
+        }
+    }
+}
+
+.product-listing-wrap .product-list {
+    padding-top: 15px;
+    .product-detail {
+        .product-box {
+            padding-top: 20px;
+            padding-bottom: 10px;
+            border-bottom: 1px dashed $plp_item_bg_color;
+            margin-bottom: 40px;
+            .product-pricing {
+                margin-top: 15px;
+                text-align: right;
+                position: relative;
+                span {
+                    font-size: 75%;
+                    position: absolute;
+                    left: 0;
+                    color: initial;
+                    font-size: 12px;
+                    top: 50%;
+                    transform: translateY(-50%);
+                }
+                a {
+                    padding: 7px 30px;
+                    display: inline-block;
+                }
+            }
+            .description-box {
+                overflow: hidden;
+                height: 100px;
+                background-color: $plp_item_bg_color;
+                padding: 8px 8px;
+                font-size: 70%;
+                min-height: 100px;
+            }
+            h3 {
+                font-size: 13px;
+                min-height: 80px;
+                font-weight: bold;
+            }
+            .buy-now-btn {
+                display: none;
+                align-items: flex-start;
+                text-align: center;
+                padding: 10px;
+                background: $color_red;
+                a {
+                    color: $color_white;
+                }
+                @media screen and (max-width: 1024px) {
+                    display: block;
+                }
+            }
+        }
+    }
+}
+
+.button-small {
+    font-size: 65%;
+    padding: 10px;
+    background: $color-red;
+    color: $color_white;
+}
+
+.button-large {
+    font-size: 80%;
+    padding: 10px;
+    background: $color-red;
+    color: $color_white;
+    border: none;
+    width: 100%;
+    margin-top: 20px;
+}
+
+.width-100 {
+    width: 100%;
+}
+
+.plp-section {
+    padding: 40px 0;
+}
+
+.wrapper {
+    width: 85%;
+    margin: 0 auto;
+}
+
+
+/* Product Category Menu */
+
+
+/* [HAMBURGER - HIDDEN ON BIG SCREENS] */
+
+#page-nav label,
+#page-nav #product-hamburger {
+    display: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+@media screen and (min-width: 991px) and (max-width:1024px) {
+    .product-listing-wrap .product-list .product-detail .product-box {
+        h3 {
+            font-size: 90%;
+            min-height: 100px;
+        }
+        .product-pricing a {
+            padding: 7px 20px;
+        }
+    }
+}
+
+.wrapper {
+    width: 85%;
+    margin: 0 auto;
+    @media screen and (max-width: 1024px) {
```

Comment:
@shilpaRai123 Your CSS has all different type of patterns, doesn't look like written by one single developer or you have deliberately paid no attention to it.